server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -14,7 +14,45 @@ const MaxFetchSize = 2000;
 const MaxRetryAttempts = 3;
 const InitialRetryIntervalInMs = 1000;
 const errorSanitizationMessage = "REDACTED";
-const errorJsonKeyToSanitize = "data";
+const errorResponseKeysAllowList = [
+    "_id",
+    "clientId",
+    "clientSequenceNumber",
+    "code",
+    "codeName",
+    "documentId",
+    "driver",
+    "err",
+    "errmsg",
+    "errorDetails",
+    "errorLabels",
+    "expHash1",
+    "index",
+    "insertedIds",
+    "message",
+    "minimumSequenceNumber",
+    "mongoTimestamp",
+    "name",
+    "nInserted",
+    "nMatched",
+    "nModified",
+    "nRemoved",
+    "nUpserted",
+    "ok",
+    "op",
```

Comment:
Do we need to include all these keys? Or would it be ok to just list those needed for the error handling in the facade, like
- err
- errmsg
- errorDetails
- errorLabels
etc?

Reply:
I agree. Including keys like op, operation etc, is risky. Let's say FF changes their contracts, then we might accidentally expose CC again. Let's only log errors and tenantId/docId if that's not already part of telemetryProperties.